client/components/destination/index.tsx

```diff
@@ -86,7 +86,7 @@ const DestinationHeader = ({ geoJSON, selectedSearchResult }) => {
  * @param {Object} geoJSON - The GeoJSON data.
  * @return {JSX.Element|null} The WeatherCard component if weatherObject and weatherWeek are truthy, otherwise null.
  */
-const WeatherData = ({ geoJSON }) => {
+export const WeatherData = ({ geoJSON }) => {
```

Comment:
this is for destinations, we should be able to just get the weather card already used working